tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -325,7 +170,7 @@ def test_wrapper(test_func, *args, **kwargs):
 
 @ldap_wrapper
```

Comment:
Defining and ldap cleanup fixture (as discussed above) would remove the need for this wrapper/decorator. Pytest already has the framework to execute the method with the pattern here - without having to define and use this decorator.

Reply:
Okay got you here! ignore my above comment